pages/dkp/kommander/2.3/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -25,7 +25,7 @@ This section describes how to upgrade your Kommander Management cluster and all
   Download the Kommander application definitions:
 
   ```bash
-  wget "https://downloads.d2iq.com/dkp/v2.2.0/kommander-applications-v2.2.0.tar.gz"
+  wget "https://downloads.d2iq.com/dkp/v2.2.0/kommander-applications-v2.2.0.tar"
```

Comment:
@mmcelroy91 made me realize, we could also start changing the versioning numbers as well? What do you think?
```suggestion
  wget "https://downloads.d2iq.com/dkp/v2.3.0/kommander-applications-v2.3.0.tar"
```

Reply:
yep, agree!